src/main/java/org/influxdb/dto/Point.java

```diff
@@ -385,4 +387,51 @@ private StringBuilder formatedTime() {
     return sb;
   }
 
+  public Point tag(final String tagName, final String value) {
+
```

Comment:
I'm not disagreeing with the coding standard; I'm asking if there is a tool that will do the formatting according to the standard.  Seriously I'm trying to satisfy what you're asking for as I'm not familiar with tools compatible with checkstyle and whether they will fix the code in question.